packages/@react-spectrum/table/stories/CRUDExample.tsx

```diff
@@ -113,13 +113,13 @@ export function CRUDExample() {
           }
         </TableBody>
       </Table>
-      <Modal isOpen={editingItem} onClose={() => setEditingItem(null)}>
-        <EditDialog 
+      <Modal isOpen={!!editingItem} onClose={() => setEditingItem(null)}>
```

Comment:
curious about this change?

Reply:
isOpen expects a boolean, editingItem is an object, !! makes it a boolean and then props stops complaining :)